src/openrct2-ui/input/Shortcuts.cpp

```diff
@@ -916,6 +911,25 @@ void ShortcutManager::RegisterDefaultShortcuts()
                 context_open_window(WC_DEBUG_PAINT);
             }
         }
-    });
+    })
+    }
+};
+    
+void ShortcutManager::RegisterDefaultShortcuts()
+{
     // clang-format on
```

Comment:
This is weird to have in the middle of a function. Move this up to right after the vector definition, or even remove it entirely if clang-format does good formatting already. Try not to disable clang-format unless really necessary. Often, you can get lists to one entry per line by adding a trailing comma, and pack them together by not having the trailing comma.